README.md

```diff
@@ -25,32 +25,45 @@ documentation.
 
 Usage
 -----
-#### 1. Get the code:
+###Get the code:
 Fork and clone this repository. Issue pull-requests one document at a time.
 
 ### Running Locally (Optional)
-#### 2. Get composer:
+------------
+
+### Option 1: Use Vagrant (preferred)
+1. Install requirements:
+* [virtualBox](https://www.virtualbox.org/wiki/Downloads) >= 4.3.x
+* [ansible](http://docs.ansible.com/ansible/intro_installation.html#installing-the-control-machine) >= 1.8.x
+* [vagrant-hostmanager](https://github.com/smdahlen/vagrant-hostmanager)
+2. From inside the project root, run `vagrant up`
+3. You will be prompted for the administration password on your host machine. Obey.
+4. Visit <http://docs.local:8000/docs> in your browser of choice.
+
```

Comment:
@craychee I'm not able to complete using Option 1

I get the following output from `vagrant up`: https://gist.github.com/rachelwhitton/6050a769a520481dc746#file-vagrant_up

Virtualbox version 5.0.2
ansible 1.9.4
vagrant-hostmanager 1.6.1

Any tips?